src/utilcode/longfilepathwrappers.cpp

```diff
@@ -1366,12 +1366,26 @@ HRESULT LongFile::NormalizePath(SString & path)
         }
     }
 
-
-    //wcscpy_s always termintes with NULL, so we are saving the character that will be overwriiten
-    WCHAR temp = buffer[prefix.GetCount()];
-    wcscpy_s(buffer, prefix.GetCount() + 1, prefix.GetUnicode());
-    buffer[prefix.GetCount()] = temp;
-    path.CloseBuffer(ret + prefixLen);
+	SString fullpath(SString::Literal, buffer + prefixLen);
+
+	if (fullpath.BeginsWith(UNCPathPrefix) && prefixLen != prefix.GetCount() - 2)
```

Comment:
PowerShell allows you to set working directory to a UNC path (simply do `cd \\foo\bar`), even though cmd.exe doesn't allow it.